sqlobject/src/main/java/org/jdbi/v3/sqlobject/statement/SqlQuery.java

```diff
@@ -58,7 +61,15 @@ public Handler buildHandler(Class<?> sqlObjectType, Method method) {
 
         @Override
         void configureReturner(Query q, SqlObjectStatementConfiguration cfg) {
-            cfg.setReturner(() -> magic.map(q, q.getContext()));
+            cfg.setReturner(() -> {
+                StatementContext ctx = q.getContext();
+                Type elementType = magic.elementType(ctx);
+                UseRowMapper useRowMapper = getMethod().getAnnotation(UseRowMapper.class);
+                ResultIterable<?> iterable = useRowMapper == null
```

Comment:
Ugh, the compiler system won't tolerate the `ResultIterable<?>` return type on the lambda. I give up.